cmd/bootnode/main.go

```diff
@@ -11,13 +11,31 @@ import (
 	"github.com/ethereum/go-ethereum/p2p/discv5"
 )
 
+type nursery []*discv5.Node
+
+func (f *nursery) String() string {
```

Comment:
Why implement `fmt.Stringer` here, for logging? Could it make sense in this case to also output the node IDs? Just asking.

Reply:
this is a part of the flag.Value interface. it can be called only with zero value (from doc) and used to describe default value as a string